crates/bevy_math/src/ops.rs

```diff
@@ -227,6 +227,38 @@ mod std_ops {
     pub fn atanh(x: f32) -> f32 {
         f32::atanh(x)
     }
+
+    /// Calculates the least nonnegative remainder of `x (mod y)`.
+    ///
+    /// Precision is specified when the `libm` feature is enabled.
+    #[inline(always)]
+    pub fn rem_euclid(x: f32, y: f32) -> f32 {
+        f32::rem_euclid(x, y)
+    }
+
+    /// Computes the absolute value of x.
+    ///
+    /// Precision is specified when the `libm` feature is enabled.
+    #[inline(always)]
+    pub fn abs(x: f32) -> f32 {
+        f32::abs(x)
+    }
+
+    /// Returns the square root of a number.
+    ///
+    /// Precision is specified when the `libm` feature is enabled.
```

Comment:
f32::sqrt is deterministic and precise apparently

> Precision
> The result of this operation is guaranteed to be the rounded infinite-precision result. It is specified by IEEE 754 as squareRoot and guaranteed not to change.

Source:
https://doc.rust-lang.org/std/primitive.f32.html#method.sqrt

Double check the other operators if you don't mind

Reply:
Good catch! I've updated the descriptions for all the `std` versions of the operations I've added. The `libm` versions I've left as-is, since their precision isn't immediately apparent to me in the `libm` documentation.